### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### `Mobile Menu`
**In this milestone.**
I and Debas have implemented the following:

> Html (I created modal tag above the container for the project body)

> Css (I created a class none for the modal section)

> Js (Debas created a function to open and close the hamburger menu)

>

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  } else {
+    menu.classList.add('invisible');
+    main.classList.remove('invisible');
+  }
+}
+
+hambutton.addEventListener('click', openMenu);
+hamburgerTwo.addEventListener('click', openMenu);
+
+// scrollspy section
```
Comment (rloterh) [inline script.js]: - [ ] OPTIONAL
There are **comments** found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const hambutton = document.getElementById('menu-harmbuger');
+const main = document.querySelector('.container');
+const menu = document.querySelector('.modal-container');
+
+const hamburgerTwo = document.getElementById('modal-harmburger');
+
+function openMenu() {
```
Comment (rloterh) [inline script.js]: - [ ] OPTIONAL
It is a nice idea to prefer `arrow functions ` over normal functions, so as to keep your _function's_ syntax simpler and cleaner.

Comment (rloterh) [inline script.js]: - [ ] Your mobile menu is not working as intended. When the hamburger menu is clicked to launch the mobile menu for the first time and then the cancel button of the mobile menu overlay is clicked, the mobile menu does not open on subsequent attempts when the hamburger menu is clicked. Kindly ensure to improve upon this aspect
- [ ] Your mobile menu is not displaying fully when the hamburger menu is clicked for the first time. Kindly ensure that the mobile menu overlay fits a particular mobile screen with size

Mobile menu overlay not displaying correctly: 
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Gabriel & Debas, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚥 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

_Kindly also consider:_
- [ ] OPTIONAL
You can improve the `Author` section of your ReadMe by removing the second author heading,  to make it look like this:

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (gabrielcoder247) [reply script.js]: Hey @rloterh
 your last statement is not clear to me can you hight more on that

Comment (rloterh) [reply script.js]: When the hamburger is clicked the popup does not fill the entire mobile screen, it only covers some portions of the mobile screen, hence your mobile popup menu is not responsive. Kindly endeavor to improve upon that aspect

**Review by mke2111 — APPROVED** [review]
### Hi @gabrielcoder247, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**